Source/WebCore/Modules/speech/SpeechSynthesisUtterance.h

```diff
@@ -82,6 +82,8 @@ class WEBCORE_EXPORT SpeechSynthesisUtterance final : public PlatformSpeechSynth
     void errorEventOccurred(const AtomString& type, SpeechSynthesisErrorCode);
     void setIsActiveForEventDispatch(bool);
 
+    bool isSpeechSynthesisUtterance() const final { return true; }
```

Comment:
This should be `private` there is no point in ever calling this function on the subclass.